Habitica/src/main/java/com/magicmicky/habitrpgwrapper/lib/api/ApiService.java

```diff
@@ -100,6 +100,9 @@
     @GET("tasks/user")
     Observable<HabitResponse<TaskList>> getTasks();
 
+    @GET("tasks/user")
+    Observable<HabitResponse<ArrayList<Task>>> getUserTasks();
```

Comment:
I guess I thought at first thats easier to use the ArrayList<Task>than a TaskList (which has all Tasks in a HashMap) , gonna revert this again^^